#!/usr/bin/env python
PACKAGE = "ros_recognizer"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("uniform_radius",    double_t,  0, "Downsampling radius (default 0.01)", .02, 0, .5)
gen.add("use_integral_image",  bool_t,  0, "Use integral image normal estimation", False)
gen.add("omp_threads",          int_t,  0, "Thread pool size (default 8)", 8, 0, 20)
gen.add("descr_rad",         double_t,  0, "Features radius (default 0.03)", 0.03, 0, .5)

exit(gen.generate(PACKAGE, "ros_recognizer", "Describer"))